src/constants/course.js

```diff
@@ -1,25 +1,4 @@
 // Constants related to courses
-export const COURSE_MODES = {
-  VERIFIED: 'verified',
-  PROFESSIONAL: 'professional',
-  NO_ID_PROFESSIONAL: 'no-id-professional',
-  AUDIT: 'audit',
-  HONOR: 'honor',
-  EXECUTIVE_EDUCATION: 'executive-education',
-  PAID_EXECUTIVE_EDUCATION: 'paid-executive-education',
-  UNPAID_EXECUTIVE_EDUCATION: 'unpaid-executive-education',
-  EXECUTIVE_EDUCATION_2U: 'executive-education-2u',
-  CREDIT_VERIFIED_AUDIT: 'credit-verified-audit',
-  VERIFIED_AUDIT: 'verified-audit',
-};
-
-export const EXECUTIVE_EDUCATION_COURSE_MODES = [
-  COURSE_MODES.EXECUTIVE_EDUCATION,
-  COURSE_MODES.PAID_EXECUTIVE_EDUCATION,
-  COURSE_MODES.UNPAID_EXECUTIVE_EDUCATION,
-  COURSE_MODES.EXECUTIVE_EDUCATION_2U,
-];
```

Comment:
Removed duplicate constants that existed at the `src` level directory. Refactored usage of `COURSE_MODES` to `COURSE_MODES_MAP`